src/core/packages/saved-objects/api-server-internal/src/lib/apis/update.ts

```diff
@@ -194,8 +194,7 @@ export const executeUpdate = async <T>(
       refresh,
       document: rawUpsert._source,
       ...(version ? decodeRequestVersion(version) : {}),
-      // @ts-expect-error
-      require_alias: true,
+      querystring: { require_alias: true },
```

Comment:
@JoshMock, here's a missing query parameter in the spec.

Reply:
Thanks, I opened https://github.com/elastic/elasticsearch-specification/pull/3954 to add it (plus a bunch of others that were also missing).